class.photon.php

```diff
@@ -422,9 +421,30 @@ public function filter_image_downsize( $image, $attachment_id, $size ) {
 			 * 	 @type $size Image size. Can be a string (name of the image size, e.g. full) or an integer.
 			 * }
 			 */
-			apply_filters( 'jetpack_photon_override_image_downsize', false, compact( 'image', 'attachment_id', 'size' ) )
-		)
+			false === apply_filters( 'jetpack_photon_admin_allow_image_downsize', false, compact( 'image', 'attachment_id', 'size' ) )
```

Comment:
Doesn't this mean that Photon URLs would then be saved in the database for each post includes images? That would make it difficult for one to move away from Photon later on, as it would require finding and replacing all Photon URLs in posts.